src/main/java/seedu/calendar/AddSecondSemester.java

```diff
@@ -0,0 +1,20 @@
+package seedu.calendar;
+
+import seedu.command.Command;
+import seedu.event.EventList;
+import seedu.exception.DukeException;
+
+public class AddSecondSemester extends Command {
```

Comment:
What is the difference between addfirstsemester and addsecondsemester? Do the prof need the distinction between them?

Reply:
Or can it be a year long thing?